modules/nodes-nodes-working-setting-booleans.adoc

```diff
@@ -0,0 +1,56 @@
+[id="nodes-nodes-working-setting-booleans"]
+
+= Setting SELinux booleans
+
+{product-title} allows you to enable and disable a SELinux boolean on a {op-system-first} node. The following procedure explains how to modify SELinux booleans on nodes using the Machine Config Operator (MCO). The `container_manage_cgroup` is the example sebool in this procedure. You can modify this value to whichever sebool you need.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+
+.Procedure
+
+. Create a new file with a `MachineConfig` object displayed in the following example:
+[source, terminal]
++
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-setsebool
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Set SELinux booleans
+          Before=kubelet.service
+
+          [Service]
+          Type=oneshot
+          ExecStart=/sbin/setsebool container_manage_cgroup=on
+          RemainAfterExit=true
+
+          [Install]
+          WantedBy=multi-user.target graphical.target
+        enabled: true
+        name: setsebool.service
+----
++
+
+. Apply the new `MachineConfig` object to the bluser by running the following command:
```

Comment:
I'm not sure what "bluser" is here? Consider "Create the new `MachineConfig` object by running the following command:"